test-app/frontend/package.json

```diff
@@ -19,8 +19,12 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "test": "cross-env CI=true react-scripts test --passWithNoTests",
+    "eject": "react-scripts eject",
+    "prepare": "cd ../../ && husky install ./test-app/frontend/.husky",
+    "lint-check": "eslint 'src/**/*.{ts,tsx}' --color",
+    "lint-fix": "eslint --fix 'src/**/*.{ts,tsx}' --color",
```

Comment:
Can this just be `src/**.{ts,tsx}`?

Reply:
I think the difference between src/**.{ts,tsx}? and src/**/**.{ts,tsx}? is that  src/**/**.{ts,tsx}? will recognize files in sub-directories, so for now I am leaving it as is

Reply:
Lol I think this got messed up since `**` is for bolding in markdown. If you put the text in backticks \`, you can display it normally how you want. The reason I suggested `src/**.{ts,tsx}` is because I am worried that the `tsx/ts` files like `App.tsx` won't be linted if it's `src/**/*.{ts,tsx}`. Perhaps that is not correct, though.